app/services/encryptor_service.rb

```diff
@@ -0,0 +1,15 @@
+class EncryptorService
+  def initialize(
+    key = Rails.application.secrets.secret_key_base.bytes[0..31].pack('c' * 32)
+  )
+    @crypt = ActiveSupport::MessageEncryptor.new(key, digest: 'base64_urlsafe')
```

Comment:
`MessageEncryptor` is not _url_safe_ in Rails Versions < 7


Use a replacement for `MessageEncryptor` or upgrade rails version to 7.1